filebeat/module/traefik/access/test/test.log-expected.json

```diff
@@ -5,30 +5,22 @@
         "event.module": "traefik", 
         "input.type": "log", 
         "log.offset": 0, 
-        "traefik.access.body_sent.bytes": "0", 
+        "traefik.access.body_sent_bytes": "0", 
         "traefik.access.http_version": "1.1", 
         "traefik.access.method": "GET", 
-        "traefik.access.referrer": "http://example.com/login", 
         "traefik.access.remote_ip": "192.168.33.1", 
         "traefik.access.response_code": "304", 
         "traefik.access.url": "/ui/favicons/favicon-16x16.png", 
-        "traefik.access.user_agent.device": "Other", 
```

Comment:
It seems the user_agent is not parsed anymore?

Reply:
Fixed, thanks!